### PR TITLE
Fix font and spacing in node selection menu

### LINKED ISSUE #139: Menu for selecting nodes has incorrect font and spacing in mobile

### PR DESCRIPTION
**Pull request by alexandercyu**
Fixes #139 
- Fix font and spacing between node selection menu items
- Change space between node selection menu items to 50px in tablet size

### CODE REVIEW

File: static/css/marble.css [hunk 1]
```diff
@@ -1096,7 +1096,40 @@ h6{
         display: inline-block;
         text-decoration: unset;
     }
-  
+
+    /*Node*/
+    .margin-node-menu-item {
```
Comment (mishaschwartz) [inline static/css/marble.css:1101]: identical in all media widths

**Review by mishaschwartz — CHANGES_REQUESTED** [review]

**Comment by alexandercyu** [comment]
Moved repeated class out of media query.

**Review by ShrutiK100 — APPROVED** [review]

**Review by mishaschwartz — CHANGES_REQUESTED** [review]
There is currently no space between the node menu items on `sm` size

**Comment by alexandercyu** [comment]
> There is currently no space between the node menu items on `sm` size

Fixed the spacing between node menu items on sm size.

Cleaned up" >=576px" media query.

**Review by mishaschwartz — APPROVED** [review]
This looks good.

It's a little difficult to review all the of the code cleanup since a lot of that doesn't seem related to this PR but as far as I can tell by looking at the UI it doesn't seem to break anything.

**Comment by alexandercyu** [comment]
> This looks good.
> 
> It's a little difficult to review all the of the code cleanup since a lot of that doesn't seem related to this PR but as far as I can tell by looking at the UI it doesn't seem to break anything.

The cleanup was mostly to get rid of duplicates in the " >=576px" media query.  The differences I remember are the space between the node page heading and value statement, and the padding in the node services cards.

**Pull request merged**